docs/development.md

```diff
@@ -333,3 +334,8 @@ In order to test the latest stable released version (tagged as `latest`) on
 your kubernetes cluster, follow the
 [automated](userguide.md#automated-deployment) or
 [manual](userguide.md#manual-deployment) instructions from the user guide.
+
+## Updating Document
+
+If you are going to add some new sections for the document, make sure to update the toc.
```

Comment:
Hmm, sorry I missed this earlier but I think 'Table of Contents' would be preferable to 'toc' so there's no confusion.

Reply:
@marun Done, but I was using 'table of contents' instead, hope it is OK.